app/TracingStrategyAssets.js

```diff
@@ -0,0 +1,80 @@
+import { Images } from './assets';
+import { config } from './COVIDSafePathsConfig';
+import languages from './locales/languages';
+
+export const isGPS = config.tracingStrategy === 'gps';
+
+// Onboarding2
+export const onboarding2BackgroundImage = isGPS
+  ? Images.LaunchScreen2
+  : Images.LaunchScreen2BT; 
+export const onboarding2HeaderText = isGPS
+  ? languages.t('label.launch_screen2_header_location')
```

Comment:
Generally `languages.t` is being phased out because its not dynamic especially in this exact use case.

Because these `t(...)` are all called outside of a component, they are fixed to the language at module load time.

Need to find a way to make the `t()` dynamic at usage/render time otherwise language changes in Settings.js will not be reflected dynamically.

Its **not pretty** but perhaps this module can export a _function_ that takes the `t` value:

```js
import { getAppSpecificAssets } from '../TracingStrategyAssets';

const {t} = useTranslation();
const { aboutHeaderText } = getAppSpecificAssets(t);
```

Reply:
This looks good to me!

Reply:
Thanks - I decided to make the TracingStrategyAssets a function so I could import and use the useTranslation hook. Let me know your thoughts about these changes. Thanks!